iot/iot-device-registry-infinispan/src/main/java/io/enmasse/iot/registry/infinispan/util/Credentials.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.iot.registry.infinispan.util;
+
+import static io.vertx.core.json.Json.decodeValue;
+import static java.util.Arrays.asList;
+import static java.util.Arrays.stream;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.hono.service.management.credentials.CommonCredential;
+import org.eclipse.hono.service.management.credentials.GenericCredential;
+import org.eclipse.hono.util.Constants;
+import org.eclipse.hono.util.CredentialsConstants;
+
+import io.enmasse.iot.registry.infinispan.device.data.DeviceCredential;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public final class Credentials {
+
+    private Credentials() {}
```

Comment:
Maybe rename this class to CredentialsHelper and remove constructor?

Reply:
I think the Java "convention" is to write helper classes named Foo_s_:

* `java.util.Collections`
* `java.util.Arrays`
* `java.nio.file.Files`

Also should helper classes, as long as they have only static methods,  be `final` and their constructor `private`, in order to prevent people from creating instances. Removing the constructor would make the default constructor accessible.

* https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/design/HideUtilityClassConstructorCheck.html

Reply:
OK. It confused me for a bit, as I was thinking it actually contains credentials. Sorry about constructor remark, I missed it was private (so wanted to prevent folks to use it)